project/plugins.sbt

```diff
@@ -1,16 +1,16 @@
 logLevel := Level.Warn
 
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.18")
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0-M8")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.2")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.13")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.1")
```

Comment:
This can still be upgraded

Reply:
I can upgrade sbt-protoc to 1.0.6, but according to the repos MLeap looks into, those versions for sbt-sonatype and sbt-native-packager are the latest I can get for scala 2.12/sbt-1.0:

https://scala.jfrog.io/ui/native/sbt-plugin-releases/com.typesafe.sbt/sbt-native-packager/scala_2.12/sbt_1.0/
https://repo1.maven.org/maven2/com/thesamet/sbt-protoc_2.12_1.0/
https://repo1.maven.org/maven2/org/xerial/sbt/sbt-sonatype_2.12_1.0/

Reply:
Sorry, I should miss something.
Where are "the repos MLeap looks into" defined ?
SNT code repo redirects to this artifact repo for the list of their release.
https://index.scala-lang.org/sbt/sbt-native-packager/artifacts/sbt-native-packager

Anyway, this is not a blocking comment, it is not directly linked to the goal of the PR.

Reply:
Not sure where they are defined, but if you define a version that doesn't exist, the error message tells you which repos its looking into:
```
...
[error] sbt.librarymanagement.ResolveException: Error downloading com.thesamet:sbt-protoc;sbtVersion=1.0;scalaVersion=2.12:1.0.8
[error]   Not found
[error]   Not found
[error]   not found: https://repo1.maven.org/maven2/com/thesamet/sbt-protoc_2.12_1.0/1.0.8/sbt-protoc-1.0.8.pom
[error]   not found: /Users/mtrottiermcdonald/.ivy2/localcom.thesamet/sbt-protoc/scala_2.12/sbt_1.0/1.0.8/ivys/ivy.xml
[error]   not found: https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.thesamet/sbt-protoc/scala_2.12/sbt_1.0/1.0.8/ivys/ivy.xml
[error]   not found: https://repo.typesafe.com/typesafe/ivy-releases/com.thesamet/sbt-protoc/scala_2.12/sbt_1.0/1.0.8/ivys/ivy.xml
...
```